src/main/scala/org/dia/apps/AccumulationsApp.scala

```diff
@@ -0,0 +1,125 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.dia.apps
+
+import java.io.{ File, PrintWriter }
+
+import scala.collection.mutable
+import scala.io.Source
+
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.security.UserGroupInformation
+
+import org.apache.spark.rdd.RDD
+
+import org.dia.core.{SciDataset, SciSparkContext, SRDDFunctions, Variable}
+import org.dia.urlgenerators.OpenDapTRMMURLGenerator
+import org.dia.utils.{FileUtils, NetCDFUtils}
+
+object AccumulationsApp extends App {
+  val logger = org.slf4j.LoggerFactory.getLogger(this.getClass)
+  val masterURL = if (args.isEmpty) "local[*]" else args(0)
+  val partitions = if (args.length <= 1) 8 else args(1).toInt
+  val hdfsURL = if (args.length <= 2) "hdfs://localhost:9000" else args(2)
+  val varname = if (args.length <= 3) "precipitation" else args(3)
+  val outputLoc = if (args.length <= 4) "output" else args(4)
+  val credentialsFilePath = "src/test/resources/TestHTTPCredentials"
+  val credentialList = Source.fromFile(credentialsFilePath)
+    .getLines()
+    .map(p => {
+      val split = p.split("\\s+")
+      (split(0), split(1))
+    }).toList
+  val username = credentialList(0)._2
+  val password = credentialList(1)._2
+
+  val outputDir = FileUtils.checkHDFSWrite(outputLoc)
+
+  val hdfsDir = outputDir + System.getProperty("file.separator") + "TRMM3B42.txt"
```

Comment:
Since we use the function call to System.getProperty("file.separator"), it could be worth it to just assign the string to a variable named fsSepStr or fSepStr and use that throughout.